src/components/ReportActionItem/IOUPreview.js

```diff
@@ -144,9 +143,12 @@ const IOUPreview = (props) => {
     // Pay button should only be visible to the manager of the report.
     const isCurrentUserManager = managerEmail === sessionEmail;
 
+    const moneyRequestAction = ReportUtils.getMoneyRequestAction(props.action);
+
     // If props.action is undefined then we are displaying within IOUDetailsModal and should use the full report amount
-    const requestAmount = props.isIOUAction ? lodashGet(props.action, 'originalMessage.amount', 0) : ReportUtils.getMoneyRequestTotal(props.iouReport);
-    const requestCurrency = props.isIOUAction ? lodashGet(props.action, 'originalMessage.currency', CONST.CURRENCY.USD) : props.iouReport.currency;
+    const requestAmount = props.isIOUAction ? moneyRequestAction.total : ReportUtils.getMoneyRequestTotal(props.iouReport);
+    const requestCurrency = props.isIOUAction ? moneyRequestAction.currency : props.iouReport.currency;
+    const requestComment = Str.htmlDecode(moneyRequestAction.comment).trim()
```

Comment:
Fuuu why did the prettier not catch this 😡

Reply:
idk but it is creating some major weirdness for me locally. Is anyone fixing it?

Reply:
https://github.com/Expensify/App/pull/18985